android/src/main/java/ar/gov/indec/react/spatialite/SpatialiteReactPackage.java

```diff
@@ -11,19 +11,19 @@
 import java.util.Collections;
 
 public class SpatialiteReactPackage implements ReactPackage {
-    @Override
+    
     public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
```

Comment:
But the official doc says that `createViewManagers` and `createNativeModules` should be decorated with `@Override` here https://facebook.github.io/react-native/docs/native-modules-android.
Maybe this was a breaking change because `createJSModules` is not longer in the docs. We should check and upgrade the `react-native` package on the `peerDependencies`@ `package.json` to be clear of the breaking change.

Reply:
`react-native@0.47` removed `createJSModules` on Android.
We can delete the `createJSModules` completly and we should keep the `@Override` decorator on the other 2 methods.
And we should upgrade the `peerDependencies` for `react-native@>=0.47.0`